client/state/themes/actions/activate.js

```diff
@@ -29,33 +22,9 @@ export function activate(
 	siteId,
 	source = 'unknown',
 	purchased = false,
-	keepCurrentHomepage = false
+	keepCurrentHomepage = true
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
Should we pass the `persist_homepage_template: true` always as well, or not?
fbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qwfba%2Qncv%2Qnpgvir%2Qgurzr%2Qraqcbvag.cuc%3Se%3Q1p463405%2333-og

Reply:
Thanks for pointing that out. 

I missed that `persist_homepage_template` is always `false` because the  `themes/theme-switch-persist-template` feature flag is `false` in all environments. 

That means the logic related to preserving the home template is not being used. 

See pbxlJb-2X4-p2#comment-2162 and the changes in that API fbhepr%2Suvfgbel%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qwfba%2Qncv%2Qnpgvir%2Qgurzr%2Qraqcbvag.cuc-og